docs/text_handling.md

```diff
@@ -0,0 +1,41 @@
+# Support for text data format {#ovms_docs_text}
+
+OpenVINO Model Server can now greatly simplify writing the applications with Natural Language Processing models. For the use cases related to text analysis or text generation, the client application can communicate with the model server using the original text format. There is no requirement to perform pre and post processing on the client side. Tokenization and detokenization can be not fully delegated to the server.
```

Comment:
"Tokenization and detokenization can be **not** fully delegated to the server"
is that correct?